webserver-darwin.sh

```diff
@@ -26,4 +26,41 @@ else
   echo "installing nano"
   brew install nano 
 fi
+# Launch Relativepath instance
+if ( multipass list | grep relativepath | grep Running > /dev/null )
+then 
+  echo -e "\n==== Relativepath VM present ====\n"
+else 
+  echo -e "\n==== Creating Relativepath VM ====\n"
+  multipass launch --name relativepath --cloud-init cloud-init.yaml
```

Comment:
Let's wait till we get to this in the sessions!

Reply:
my bad, I actually meant to trim this down. to line 35 minus the cloud-init.yaml.